charts/selenium-grid/README.md

```diff
@@ -71,7 +71,7 @@ This table contains the configuration parameters of the chart and their default
 | `chromeNode.imageName`                  | `selenium/node-chrome`             | Image of chrome nodes                                                                                                      |
 | `chromeNode.imageTag`                   | `4.3.0-20220628`                   | Image of chrome nodes                                                                                                      |
 | `chromeNode.imagePullPolicy`            | `IfNotPresent`                     | Image pull policy (see https://kubernetes.io/docs/concepts/containers/images/#updating-images)                             |
-| `chromeNode.ports`                      | `[5553]`                           | Port list to enable on container                                                                                           |
+| `chromeNode.ports`                      | `[5555]`                           | Port list to enable on container                                                                                           |
```

Comment:
Why? This might break things for current users.

Reply:
I am aware this is not the port we have always used, but in any case it is still a valid port.

Reply:
Why this port is valid if nothing on pod really listens to it? Even why it can break anything?
It is not used anywhere else and even services only forward one another port:
https://github.com/SeleniumHQ/docker-selenium/blob/trunk/charts/selenium-grid/templates/chrome-node-service.yaml

Reply:
OK, makes sense. Can you please resolve the conflicts and then we can merge?